astropy/visualization/wcsaxes/tests/test_misc.py

```diff
@@ -531,3 +531,11 @@ def test_wcs_type_transform_regression():
 
     high_wcs = HighLevelWCSWrapper(sliced_wcs)
     ax.get_transform(sliced_wcs)
+
+
+def test_multiple_draws_grid_contours(tmpdir):
+    fig = plt.figure()
+    ax = fig.add_subplot(1, 1, 1, projection=WCS())
+    ax.grid(color='black', grid_type='contours')
+    fig.savefig(tmpdir / 'plot.png')
```

Comment:
Why the double save?

Reply:
It's what triggers the bug - because it makes the contours be removed and readded